frontend/config/routes.rb

```diff
@@ -3,9 +3,9 @@
     root to: 'home#index'
 
     resources :products, only: [:index, :show]
```

Comment:
@MatthewKennedy should we also add `as: Spree::Config[:storefront_product_path]` here? because this will only update links to related products but not the products show / index page

Reply:
Yes, I think that's a good idea, I'll get that sorted.